src/index.ts

```diff
@@ -388,9 +388,10 @@ export class GraphQLServer {
 
     return new Promise((resolve, reject) => {
       const combinedServer = server
-      const port = typeof this.options.port !== "number" 
-        ? parseInt(this.options.port) 
-        : this.options.port
+      const port =
+        typeof this.options.port !== 'number'
+          ? parseInt(this.options.port, 10)
```

Comment:
this was causing an error with the linting, so I fixed this -- let me know if I should remove this change